data/bash-completion/fwupdmgr

```diff
@@ -130,7 +131,7 @@ _fwupdmgr()
 			_show_modifiers
 		fi
 		;;
-	install)
+	install|local-install)
```

Comment:
I will fix it later.

Reply:
``` diff
diff --git a/data/bash-completion/fwupdmgr b/data/bash-completion/fwupdmgr
index 8912864b..40b8ee4d 100644
--- a/data/bash-completion/fwupdmgr
+++ b/data/bash-completion/fwupdmgr
@@ -83,6 +83,16 @@ _show_device_ids()
        COMPREPLY+=( $(compgen -W "${description}" -- "$cur") )
 }

+_show_release_versions()
+{
+       if ! command -v jq &> /dev/null; then
+               return 0
+       fi
+       local description
+       description="$(command fwupdmgr get-devices --json 2>/dev/null | jq ".Devices[] | select(.DeviceId==\"$1\") | .Releases[].Version" 2>/dev/null)"
+       COMPREPLY+=( $(compgen -W "${description}" -- "$cur") )
+}
+
 _show_remotes()
 {
        local remotes
@@ -131,7 +141,19 @@ _fwupdmgr()
                        _show_modifiers
                fi
                ;;
-       install|local-install)
+       install)
+               #device ID or modifiers
+               if [[ "$prev" = "$command" ]]; then
+                       _show_device_ids
+               #version
+               elif [[ "$prev" = "${COMP_WORDS[2]}" ]]; then
+                       _show_release_versions "${COMP_WORDS[2]}"
+               #modifiers
+               else
+                       _show_modifiers
+               fi
+               ;;
+       local-install)
                #find files
                if [[ "$prev" = "$command" ]]; then
                        _filedir

```

Reposting, in case of ;)

Reply:
My comment was pending... sorry

Reply:
I fixed it up and force pushed, thanks.